fedmsg/tests/crypto/test_utils.py

```diff
@@ -7,6 +7,74 @@
 from fedmsg.crypto import utils
 
 
+class FixDatanommerMessageTests(unittest.TestCase):
+    """Tests for the :func:`fedmsg.crypto.utils.fix_datagrepper_message` function."""
+
+    def test_no_source_keys(self):
+        """Assert messages with neither "source_name" or "source_version" are untouched."""
+        original_message = {'my': 'message'}
+
+        self.assertTrue(original_message is utils.fix_datagrepper_message(original_message))
+
+    def test_no_source_name(self):
+        """Assert messages missing the "source_version" key are untouched."""
```

Comment:
Perhaps this comment was meant to say "source_name" instead of "source_version"?